ts/Shared/Types.ts

```diff
@@ -0,0 +1,45 @@
+/* *
+ *
+ *  (c) 2010-2021 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+namespace Types {
```

Comment:
I'm not convinced do the naming, how about `SharedTypes`  or `GlobalTypes`?

Reply:
This seems to be redundant as it already is in the `Shared`. `Global` would be not accurate as it is not part of the global namespace.